pyaedt/edb_core/edb_data/obj_base.py

```diff
@@ -0,0 +1,16 @@
+class ObjBase(object):
+    """Manages EDB functionalities for a base object."""
+
+    def __init__(self, pedb, model):
+        self._pedb = pedb
+        self._edb_object = model
+
+    @property
+    def is_null(self):
+        """Flag indicating if this object is null."""
+        return self._edb_object.IsNull()
+
+    @property
+    def type(self):
+        """Get type."""
```

Comment:
Properties should be defined as nouns. For this ``type`` property, the description should be "Type of ...." (Type of the base object?) We don't need to call out GET or SET usage in the description. If needed, an "Examples" section in the docstring could provide usage examples.)